### DISCUSSION TITLE
ESP32 OPENDTU flash geht nicht, Ahoy geht

### DISCUSSION THREAD

**Discussion by Agerter** [tbnobody/OpenDTU#1423] [General]
Hallo ich hab folgendes Problem festgestellt.
Wenn ich eine Chip nehme der unter WLAN als ESP32 Accespoint erkannt wird, aber sich nach Eingabe des Passwortes sich nicht mit dem Netzwerk verbindet , komplett loesche und dann erneut mit OpenDTU flashe ,bekomme ich kein openDTU mehr im WLAN angezeigt. Zuvor wurde mir wenigstens der Accespoint noch angezeigt.

Und jetzt das komische !
wenn ich dann allerdings Ahoy auf dem Chip flashe wird AHOY im WLAN als Accespoint angezeigt und ich kann mich problemlos verbinden.
Kann es sein das es an der factory.bin und der generic.bin liegt wenn ich openDTU ein 2tesmal auf den Chip flashen moechte.??
wer kann mir da helfen ich steh vor einem Raetzel.
Danke schonmal!

**Comment by melmager** [comment]
keine ahnung ob es weiterhilft aber ich habe eben ne neue firmware hochgaladen - trotzdem kannte er er noch die alten einstellungen
vermutlich muss man den es vorher mal löschen
und wenn bitte den opendtu-factory.bin nutzen :-)
